test/PangenomeFileUtil_server_test.py

```diff
@@ -51,73 +87,57 @@ def tearDownClass(cls):
     def getWsClient(self):
         return self.__class__.wsClient
 
-    def getWsName(self):
-        if hasattr(self.__class__, 'wsName'):
-            return self.__class__.wsName
-        suffix = int(time.time() * 1000)
-        wsName = "test_PangenomeFileUtil_" + str(suffix)
-        ret = self.getWsClient().create_workspace({'workspace': wsName})
-        self.__class__.wsName = wsName
-        return wsName
-
     def getImpl(self):
         return self.__class__.serviceImpl
 
     def getContext(self):
         return self.__class__.ctx
 
-    def test_your_method(self):
-        pfut = self.getImpl()
-
-        # upload pangenome object (needs genomes, but assumes old genome type!!)
-        # skip this for now, and use a preloaded one
-        obj_name = 'pangenome'
-        ws_name =  'msneddon:1450308782878' #self.getWsName()
-        #with open(os.path.join('data','dummy_pangenome.json')) as data_file:    
-        #    pgData = json.load(data_file)
-        #ws = self.getWsClient();
-        #info = ws.save_objects({'workspace':self.getWsName(), 'objects':[
-        #            {
-        #                'type':'KBaseGenomes.Pangenome',
-        #                'data':pgData,
-        #                'name':obj_name
-        #            }]})
-        #print('saved dummy pangenome')
-        #pprint(info)
-
-        # attempt to download as tsv file
-        res = pfut.pangenome_to_tsv_file(self.getContext(),
-                                        {
-                                            'pangenome_name':obj_name,
-                                            'workspace_name':ws_name,
-                                        })
+    def test_to_tsv(self):
+        res = self.getImpl().pangenome_to_tsv_file(self.getContext(), {
+                'pangenome_name': self.obj_name,
+                'workspace_name': self.wsName,
+            })
         pprint(res)
```

Comment:
so the test will pass. But it really didn't check content of saved files. Since previous people didn't check either. I guess you can just leave it this way. But if you have time, I would check content of saved files.